docs/theming.md

```diff
@@ -271,16 +271,16 @@ The remainder of the props are spread onto the `buttonComponent` while the `smal
 
 #### Should
 
-Recomendations for components:
+Recommendations for components:
 
-* Should establish the complete HTML DOM structure necessary to operate itself, and the minimum styling needed.
-* Should declare and export all of its state-classes via `publicClassNames: true`.
-* Should allow component and possibly sub-component (child component) overrides where reasonable. 
+* Establish the complete HTML DOM structure necessary for operation, with the minimum styling needed.
+* Declare and export all of state-classes via `publicClassNames: true`.
```

Comment:
There's one too many or one too few articles in this sentence, @webOS101.
"...export all state-classes..."
or
"...export all of the state-classes..."

Reply:
dang.